components/match2/wikis/rainbowsix/match_group_input_custom.lua

```diff
@@ -72,6 +72,17 @@ function CustomMatchGroupInput.processOpponent(record, date)
 		opponent = {type = Opponent.literal, name = 'BYE'}
 	end
 
+	-- If date if epoch, resolve using tournament dates instead
+	-- Epoch indicates that the match is missing a date, but in order to get correct child team template
+	-- We need a present date and not 1970
+	if date == _EPOCH_TIME_EXTENDED then
```

Comment:
Since we are giving a new value to `date` anyway, why not just set it to `nil` on line 302?

Reply:
`date` is a local parameter and is passed by value, so that date used here is only inside this function. 
The `match.date` used from 302 is also used to eg. store into lpdb.

Reply:
Ah, then use a new var instead, it's confusing

Reply:
Updated